apps/server/src/api-data/assets/assets.service.ts

```diff
@@ -0,0 +1,31 @@
+import { publicFiles } from '../../setup/index.js';
+import { existsSync } from 'node:fs';
+import { readFile, writeFile } from 'node:fs/promises';
+
+/**
+ * Reads the user's css file
+ * @returns css contents in the file
+ */
+export async function readCssFile(): Promise<string> {
+  const path = publicFiles.cssOverride;
+  if (!existsSync(path)) {
+    throw new Error('File not found');
+  }
```

Comment:
question: if the path does not exist (ie: the user deleted the file mid session), should we write the contents of the bundled css to a new file?